java/ql/automodel/publish.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -e
+
+AUTOMODEL_ROOT="$(dirname $0)"
+WORKSPACE_ROOT="$AUTOMODEL_ROOT/../../../.."
+GRPS="automodel,-test"
+
+if [ -z "$CODEQL_DIST" ]; then
```

Comment:
(Optional)...you could also check if `gh codeql` is available and use that if `CODEQL_DIST` is unset.

Reply:
Lets defer that in case we want to make changes to the solorigate pack as well.